### PR TITLE
[samplecode][3/3]Implement regular paged callable sample code

### PR DESCRIPTION
**Pull request by summer-ji-eng**
method generated from `createCallableMethod` [here](https://github.com/googleapis/gapic-generator-java/blob/c7c8712dcd971d0ecb77f85695bdc8f967ad85ad/src/main/java/com/google/api/generator/gapic/composer/ServiceClientClassComposer.java#L485)
```
   * try (EchoClient echoClient = EchoClient.create()) {
   *   PagedExpandRequest request = PagedExpandRequest.newBuilder().build();
   *   while (true) {
   *     PagedExpandResponse response = echoClient.pagedExpandCallable().call(request);
   *     for (PagedExpandResponse element : response.getResponsesList()) {
   *       // doThingsWith(element);
   *     }
   *     String nextPageToken = response.getNextPageToken();
   *     if (!String.isNullOrEmpty(nextPageToken)) {
   *       request = request.toBuilder().setPageToken(nextPageToken).build();
   *     } else {
   *       break;
   *     }
   *   }
   * }
  ```

TODO: rebase #641 to get rid of `CloudRedisClient.java` change. (Done)

### CODE REVIEW

File: src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java [hunk 1]
```diff
@@ -549,7 +555,139 @@ public static String composeRegularCallableMethodHeaderSampleCode(
 
     List<Statement> bodyStatements = new ArrayList<>();
 
-    if (!method.isPaged()) {
+    if (method.isPaged()) {
```
Comment (miraleung) [inline src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java:558]: This is a very large block, can we extract it into a separate method?

**Review by miraleung — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java (hunk 1 above)
Comment (summer-ji-eng) [reply src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java:614]: Updated.

File: src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java [hunk 2]
```diff
@@ -964,6 +971,144 @@ public static String composeRegularCallableMethodHeaderSampleCode(
     return bodyStatements;
   }
 
+  private static List<Statement> composePagedCallableSampleCodeBodyStatements(
+      Method method,
+      VariableExpr clientVarExpr,
+      VariableExpr requestVarExpr,
+      Map<String, Message> messageTypes) {
+    List<Statement> whileBodyStatements = new ArrayList<>();
```
Comment (miraleung) [inline src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java:979]: Can we move this to the line just before it is first used?

File: test/integration/goldens/logging/ConfigClient.java [hunk 3]
```diff
@@ -395,6 +395,23 @@ public final ListBucketsPagedResponse listBuckets(ListBucketsRequest request) {
...
+   *
+   * <pre>{@code
+   * try (ConfigClient configClient = ConfigClient.create()) {
+   *   while (true) {
+   *     ListBucketsResponse response = configClient.listBucketsCallable().call(request);
+   *     for (LogBucket element : response.getResponsesList()) {
+   *       // doThingsWith(element);
+   *     }
+   *     String nextPageToken = response.getNextPageToken();
+   *     if (!String.isNullOrEmpty(nextPageToken)) {
```
Comment (miraleung) [inline test/integration/goldens/logging/ConfigClient.java:407]: `Strings.isNullOrEmpty` - this is a Guava method.

**Review by miraleung — CHANGES_REQUESTED** [review]

File: test/integration/goldens/logging/ConfigClient.java (hunk 3 above)
Comment (summer-ji-eng) [reply test/integration/goldens/logging/ConfigClient.java:407]: Match the original sample code. Here is an [example](https://github.com/googleapis/java-logging/blob/release-v1.101.3/google-cloud-logging/src/main/java/com/google/cloud/logging/v2/ConfigClient.java#L294).

Do we need to make any change here?

File: src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java (hunk 2 above)
Comment (summer-ji-eng) [reply src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java:979]: Sure. Will do.

File: test/integration/goldens/logging/ConfigClient.java (hunk 3 above)
Comment (summer-ji-eng) [reply test/integration/goldens/logging/ConfigClient.java]: Gotcha. Replace "String" by "Strings".

**Review by miraleung — APPROVED** [review]

**Pull request merged**